cmd/govim/vimstate.go

```diff
@@ -175,9 +164,15 @@ func (v *vimstate) setUserBusy(args ...json.RawMessage) (interface{}, error) {
 	var isBusy int
 	v.Parse(args[0], &isBusy)
 	v.userBusy = isBusy != 0
+
+	if err := v.updateReferenceHighlight(!v.userBusy); err != nil {
```

Comment:
As discussed on Slack, I think we need to make this call async. We need to ensure there is close to zero latency when moving from "not busy" to "busy". The tail latency of a call to `gopls` will kill this.